### PR TITLE
change phishlabs_ioc_eir to phishlabs_ioc_eir_v2

### PR DESCRIPTION
**Pull request by GalRabin**
## Status
Ready

## Description
Change integration name by @glicht request and @Arsenikr approval :
1. PhishLabsIOC --> PhishLabsIOC_EIR
2. PhishLabsIOC_EIR --> PhishLabsIOC_EIR_v2

Changed file in each integration:
```txt
Integrations/<new_Integration_name>/<new_Integration_name>.py
Integrations/<new_Integration_name>/<new_Integration_name>.yml
Integrations/<new_Integration_name>/<new_Integration_name>_description.md
Integrations/<new_Integration_name>/<new_Integration_name>_image.png
Integrations/<new_Integration_name>/<new_Integration_name>_test.py
TestPlaybooks/playbook-<new_Integration_name>_test.yml
Tests/conf.json
```

## Does it break backward compatibility?
   - Yes
       - Further details: could influence, however, I search all content repo for name references.
  

## Technical writer review
Mention and link to the files that require a technical writer review.
- [x] [CHANGELOG](https://github.com/demisto/content/blob/change_name_pishlabsioc_eir/Integrations/PhishLabsIOC_EIR_v2/CHANGELOG.md)  PhishLabsIOC_EIR_v2

### CODE REVIEW

File: Integrations/PhishLabsIOC_EIR/PhishLabsIOC_EIR.yml [hunk 1]
```diff
@@ -92,6 +92,7 @@ script:
       - closed_at
       required: false
       secret: false
+      secret: false
```
Comment (yaakovi) [inline Integrations/PhishLabsIOC_EIR/PhishLabsIOC_EIR.yml]: duplicate, please remove

File: Integrations/PhishLabsIOC_EIR/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ## [Unreleased]
-
+- change display name phishlabs eir to phishlabs eir v2
```
Comment (Itay4) [inline Integrations/PhishLabsIOC_EIR/CHANGELOG.md]: do we really need changelog here?

Comment (yaakovi) [reply Integrations/PhishLabsIOC_EIR/CHANGELOG.md]: the old Phishlabs integration is going to be called "Phishlabs EIR" (so maybe we should...)

**Comment by kirbles19** [comment]
@GalRabin Done.

**Pull request merged**